eZ/Bundle/EzPublishCoreBundle/Command/CheckURLsCommand.php

```diff
@@ -81,8 +81,8 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         );
 
         $limit = $input->getOption('iteration-count');
-        if (!ctype_digit($limit) || (int)$limit < 1) {
-            throw new RuntimeException("'--iteration-count' option should be > 0, got '{$limit}'");
+        if (!is_numeric($limit) || (int)$limit < 1) {
```

Comment:
This fix introduces a small change in accepted values... see https://stackoverflow.com/questions/236406/is-there-a-difference-between-is-int-and-ctype-digit/236411

Reply:
This happened by oversight, will fix.